api/src/test/java/org/pmiops/workbench/cdr/CdrVersionServiceTest.java

```diff
@@ -139,98 +138,109 @@ public void setUp() {
 
   @Test
   public void testSetCdrVersionDefault() {
-    when(fireCloudService.isUserMemberOfGroup(
-            user.getUsername(), registeredTier.getAuthDomainName()))
-        .thenReturn(true);
+    addMembershipForTest(registeredTier);
```

Comment:
these methods mock the fireCloudService methods (as before) and now also add/remove DbUserAccessTier entries as appropriate.  See the bottom of this file.